--- ./conductor/server/build.gradle.orig	2023-03-20 09:14:15
+++ ./conductor/server/build.gradle	2023-03-20 09:15:02
@@ -38,11 +38,13 @@
     implementation 'org.springframework.boot:spring-boot-starter-log4j2'
     implementation 'org.apache.logging.log4j:log4j-web'

-    implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-actuator', version: '2.4.13'

     implementation "org.springdoc:springdoc-openapi-ui:${revOpenapi}"

     runtimeOnly "org.glassfish.jaxb:jaxb-runtime:${revJAXB}"
+    runtimeOnly group: 'com.netflix.conductor', name: 'conductor-postgres-persistence', version: '3.13.5'
+    runtimeOnly group: 'com.netflix.conductor', name: 'conductor-metrics', version: '3.13.5'

     testImplementation project(':conductor-rest')
     testImplementation project(':conductor-common')
