--- ./conductor/build.gradle.orig	2022-12-20 14:58:26.000000000 +0000
+++ ./conductor/build.gradle	2022-12-20 14:58:46.000000000 +0000
@@ -19,7 +19,6 @@
     id 'java'
     id 'application'
     id 'jacoco'
-    id 'nebula.netflixoss' version '10.6.0'
     id 'org.sonarqube' version '3.4.0.2513'
 }
 
@@ -47,7 +46,6 @@
 apply from: "$rootDir/springboot-bom-overrides.gradle"
 
 allprojects {
-    apply plugin: 'nebula.netflixoss'
     apply plugin: 'io.spring.dependency-management'
     apply plugin: 'java-library'
     apply plugin: 'project-report'
